canary/apps/angular/angularcli/package.json

```diff
@@ -19,7 +19,7 @@
     "@angular/platform-browser-dynamic": "^14.3.0",
     "@angular/router": "^14.3.0",
     "@aws-amplify/ui-angular": "latest",
-    "aws-amplify": "latest",
+    "aws-amplify": "console-preview",
```

Comment:
`^6.0.0` doesn't resolve, we only have the custom tag to work with for now. It's not hard to change the deps, the canary apps specifically should be restored to latest before releasing anyway.

Reply:
Ah its because they aren't publishing valid semvers for 6.0, they are publishing versions like `6.0.1-console-preview.f63250a.0` can we see if they can just publish `6.0.1` to the `console-preview` tag?

Reply:
So...ask them to publish v6 early? :)

Reply:
I think we should ask if they will be publishing to 6.0 for dev preview